schemas/molecular_interaction_evidence.json

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
This schema is not consistent with the other schemas, where the indentation is 2, while here it is 4.